### DISCUSSION TITLE
Welcome to zlinux Discussions!

### DISCUSSION THREAD

**Discussion by moshix** [moshix/zlinux#1] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by juancc2007** [comment]
running Ubuntu 22.04.4 LTS

when running your zlinux installer i get the following:  

juancc2007@juanhp:~/Downloads/tkubuntu/zlinux$ ./zlinux_install.bash
./zlinux_install.bash: line 38: ./Version: No such file or directory
./zlinux_install.bash: line 66: ./logs/zLinux_installer.log.2024-04-23-14:54:09: Permission denied

Number of cores present  4. Setting Hercules to  2 
RAM in KB 7829. Setting Hercules to 4096
cp: cannot stat 'templates/hercules.cnf.template': No such file or directory
chmod: cannot access 'hercules.cnf': No such file or directory
sed: can't read hercules.cnf: No such file or directory
sed: can't read hercules.cnf: No such file or directory


    Starting zLinux installer Version  
    ===================================== 

./zlinux_install.bash: line 66: ./logs/zLinux_installer.log.2024-04-23-14:54:09: Permission denied
Please input desired zLinux disk size:  (3GB, 9GB, 27GB):  27GB
Roger,  27GB 
./zlinux_install.bash: line 66: ./logs/zLinux_installer.log.2024-04-23-14:54:09: Permission denied
./zlinux_install.bash: line 249: logs/dasdinit.log: Permission denied
Error creating DASD file. Check logs/dasdinit.log and logs/dasdinit_error.log.

please help. thank you

**Comment by wrljet** [comment]
Hi,

Could you run these commands and paste the output here:

```
ls -ld ~/Downloads/tkubuntu/zlinux
ls -l ~/Downloads/tkubuntu/zlinux
```

Bill

**Comment by juancc2007** [comment]
***@***.***:~/Downloads/tkubuntu/zlinux$ ls -ld
~/Downloads/tkubuntu/zlinux
drwxrwxr-x 4 juancc2007 juancc2007 4096 Apr 23 15:10
/home/juancc2007/Downloads/tkubuntu/zlinux

***@***.***:~/Downloads/tkubuntu/zlinux$ ls -ld
~/Downloads/tkubuntu/zlinux
drwxrwxr-x 4 juancc2007 juancc2007 4096 Apr 23 15:10
/home/juancc2007/Downloads/tkubuntu/zlinux

***@***.***:~/Downloads/tkubuntu/zlinux$

**Reply by wrljet** [reply]
That's not what I asked you for.  You ran the first command twice.

Bill

**Comment by juancc2007** [comment]
juancc2007@juanhp:~/Downloads/tkubuntu/zlinux$ ./zlinux_install.bash
./zlinux_install.bash: line 38: ./Version: No such file or directory
[sudo] password for juancc2007: 

Number of cores present  4. Setting Hercules to  2 
RAM in KB 7829. Setting Hercules to 4096
cp: cannot stat 'templates/hercules.cnf.template': No such file or directory
chmod: cannot access 'hercules.cnf': No such file or directory
sed: can't read hercules.cnf: No such file or directory
sed: can't read hercules.cnf: No such file or directory


    Starting zLinux installer Version  
    ===================================== 

Please input desired zLinux disk size:  (3GB, 9GB, 27GB):  27gb
Roger,  27GB 
Continue with 700MB ISO download? (Y/N): y

./zlinux_install.bash: line 268: ./scripts/getiso: No such file or directory
juancc2007@juanhp:~/Downloads/tkubuntu/zlinux$

**Comment by juancc2007** [comment]
sorry
here we go.

***@***.***:~/Downloads/tkubuntu/zlinux$ ls -ld
~/Downloads/tkubuntu/zlinux
drwxrwxr-x 4 juancc2007 juancc2007 4096 Apr 23 15:10
/home/juancc2007/Downloads/tkubuntu/zlinux

***@***.***:~/Downloads/tkubuntu/zlinux$ ls -l
~/Downloads/tkubuntu/zlinux
total 20
drwxrwxrwx 2 juancc2007 juancc2007  4096 Apr 26 13:52 dasd
drwxrwxrwx 2 juancc2007 juancc2007  4096 Apr 26 13:50 logs
-rwxrwxrwx 1 juancc2007 juancc2007 11717 Apr 23 14:38 zlinux_install.bash

***@***.***:~/Downloads/tkubuntu/zlinux$

**Comment by wrljet** [comment]
OK, you don't have the full project.  Quite a bit is missing.

Try to clone the full repo:

```
git clone https://github.com/moshix/zlinux.git
```

Then you should see a lot more files, for example:

```
bill@pop-os:~$ ls -1 zlinux
Changelog
cleanup_after_successful_install.bash
dasd
herc4x
hercules.cnf
hercules.rc
install
logs
README.md
run_zlinux.bash
scripts
SECURITY.md
templates
Version
zlinux_install.bash
```

Bill

**Comment by juancc2007** [comment]
Works great .., thank you.,, a note should be made on GitHub to clone the
repo before you begin.

Thank you.
